test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -57,121 +58,177 @@ call_log_manager::api_map_setup()
     _api_map["set_timestamp"] = api_method::set_timestamp;
 }
 
+session_simulator *
+call_log_manager::get_session(const std::string &session_id)
+{
+    /* session_id should not be empty. */
+    assert(!session_id.empty());
+
+    /* session_id should exist in the map. */
+    assert(_session_map.find(session_id) != _session_map.end());
+
+    /* Get the session from the session map. */
+    return (_session_map.at(session_id));
```

Comment:
We should make sure the `session` is not a `nullptr`:
```suggestion
    session_simulator *session = _session_map.at(session_id);
    assert(session != nullptr);
    return session;
```